docs/dictionary/function/libUrlFormData.lcdoc

```diff
@@ -2,7 +2,7 @@ Name: libURLFormData
 
 Type: function
 
-Syntax: libURLFormData(<key_1>, <value_1>, ..., <key_n>, <value_n>)
+Syntax: libURLFormData(<key1>, <value1> [, <key2>, <value2> ...])
```

Comment:
shouldn't this be 
```
libURLFormData(<key1>, <value1> [, <key2>, <value2> [, ...]])
```
for consistency with the other ones?

Reply:
Hmm... I'm not sure as `[, ...]` to me means repeat the last thing with a comma... where there's more complex repetitions such as here and `or type <type> ...` I think I it should wrap the whole thing

Reply:
But then the `or` example to be consistent with this should be
```
ask file <prompt> [with <defaultFilePath>] [with type <types> [or type <types>...]] [as sheet]
```

Reply:
ah... yes it should... I should revert those two I think... woops. Or do you think I'm wrong here and right there ;-)

Reply:
I don't know!

Reply:
Actually, is there not some kind of notation for `(this block can appear 0 or more times)`

Reply:
Was this ever consistent? For LCB syntax definitions we use
```
{ <thing> } -- repeat 0 or more times
{ <thing> , <separator> } -- repeat 0 or more times, separating with <separator> }
```
What do we want '...' to mean? :)